.stylelintrc

```diff
@@ -0,0 +1,109 @@
+{
+  "rules": {
+    "at-rule-empty-line-before": "always",
+    "at-rule-name-case": "lower",
+    "at-rule-name-space-after": "always",
+    "at-rule-no-unknown": true,
+    "at-rule-no-vendor-prefix": true,
+    "at-rule-semicolon-newline-after": true,
+    "block-closing-brace-empty-line-before": "never",
+    "block-closing-brace-newline-after": "always",
+    "block-closing-brace-newline-before": "always",
```

Comment:
Do we want this? And isn't this contradicting this rule? `block-closing-brace-empty-line-before` Maybe I am misunderstanding them